### PR TITLE
Change penetration aux output for lack of projection

### PR DESCRIPTION
**Pull request by recuero**
Refs. #24122.

### CODE REVIEW

**Review by lindsayad — DISMISSED** [review]

**Comment by recuero** [comment]
Thanks for looking at this! Yeah, I need to update a good set of framework tests. Will do shortly.

**Review by lindsayad — APPROVED** [review]
Let's wait until we get a next -> devel merge, and then we can ask for the green light to merge this

**Pull request merged**
